actions-job.php

```diff
@@ -1,4 +1,34 @@
- <?php include_once('header.php'); // Include Header once ?>
+ <?php include_once('header.php'); // Include Header once
+ 
+ if (isset($_POST['new_job'])) {
+     if (isset($_SESSION['is_company'])) { // Check if the key exist in $_SESSION
+         if ($_SESSION['is_company'] = true) { // Check if the log is company
```

Comment:
You have a syntax error.
```suggestion
         if (true === $_SESSION['is_company']) { // Check if the log is company
```